share/static/html/403.html

```diff
@@ -0,0 +1,40 @@
+{% extends "page.html" %}
+
+{% block title %}
+Everware
+{% endblock %}
+
+{% block stylesheet %}
+    <link rel="stylesheet" href="{{ static_url("css/style.min.css") }}" type="text/css"/>
+    <style type="text/css"></style>
+{% endblock %}
+
+{% block main %}
+
+<div id="login-main">
+<header class="jumbotron masthead" style="padding: 1px">
+    <h1 style="font-size: 55px">everware</h1>
+    <p style="font-size: 26px">Sharing is caring</p>
+</header>
+
+<div class="container">
+  <div class="row row-centered">
+    <div class="col-sm-8">
+      <h1> 403 Forbidden </h1>
+      <p>
+      You are not in a whitelist of this server. Please, contact to system administator to gain access.
```

Comment:
"Your username is not whitelisted on this server. Please contact the system administrator."